src/arkit.cs

```diff
@@ -832,26 +832,50 @@ interface ARFaceGeometry : NSCopying {
 		[Export ("vertexCount")]
 		nuint VertexCount { get; }
 
+#if !XAMCORE_4_0
+		[Obsolete ("Use the 'GetVertices' method instead.")]
 		[Export ("vertices")]
+		[Sealed] // Just to avoid the duplicate selector error
```

Comment:
that's a breaking change as it will remove `virtual`

since it's _largely_ unusable just remove the `[Export]`, which means moving it to `ARCompat.cs`, so the _right_ thing can still be done on the new API.

Reply:
Ah makes sense for `[Sealed]` an other option would have been to add `[Sealed]` to the new duplicated `[Export]` (looks like it's what we usually do). However I like the `ARCompat.cs` option better. Going for that.